lib/ansible/galaxy/role.py

```diff
@@ -207,10 +207,73 @@ def fetch(self, role_data):
                 temp_file.close()
                 return temp_file.name
             except Exception as e:
-                display.error(u"failed to download the file: %s" % to_text(e))
+                display.error("failed to download the file: %s" % to_text(e))
 
         return False
 
+    @staticmethod
+    def _is_path_sane(path: str) -> bool:
```

Comment:
try using py2-compatible comment-based type annotations like I did in https://github.com/ansible/ansible/pull/72591

Reply:
we need to keep py2 compat until we start working on ansible-core v2.12

Reply:
Will just skip it until py3 is fully here then.